base/linalg/dense.jl

```diff
@@ -278,9 +278,42 @@ function rcswap!{T<:Number}(i::Integer, j::Integer, X::StridedMatrix{T})
     end
 end
 
+function logm{T}(A::StridedMatrix{T})
```

Comment:
Unless you're using it 1) in the function body, or 2) to filter the types for which this method applies, then yeah, it's not necessary.